src/main/java/svydovets/util/NameResolver.java

```diff
@@ -58,6 +83,31 @@ public static String resolveBeanName(Method initMethod) {
         }
     }
 
+    /**
+     * This method helps to define request field name of declared parameter
+     *
+     * @param parameter set by customer field in request
+     * @return name of parameter
+     * @see PathVariable
+     * @see RequestParam
+     */
+    public static String resolveRequestParameterName(Parameter parameter) {
```

Comment:
Should use this method?

Reply:
https://github.com/Svydovets-Bobocode-Java-Ultimate-3-0/Bring/pull/47/files#diff-f815d64243cbe02236bbfeb301b099fde92912db2a4a7cb127bd14fb604eddbfR71
Resolved